test/integration_test.exs

```diff
@@ -193,6 +181,32 @@ defmodule AvalancheIntegrationTest do
     # end
   end
 
+  describe "run/4 multi-statement" do
+    setup do
+      options = test_options()
+      [options: options]
+    end
+
+    test "handles multi-statement requests", c do
+      assert {:ok, %Avalanche.Result{statement_handles: statement_handles} = result} =
+               Avalanche.run("begin transaction;select ?;select ? as two;commit;", [1, 2], [], c.options)
+
+      assert result.status == :complete
+      assert length(statement_handles) == 4
+
+      [sh1, sh2, sh3, sh4] = statement_handles
```

Comment:
If you need the results from one or more of the statements, you need to know which one it was in the order, get that statement handle, and then call `status` to get the results.